assets/src/edit-story/components/inspector/design/useDesignPanels.js

```diff
@@ -105,17 +105,28 @@ function useDesignPanels() {
     };
   }, [onSetProperties, registerSubmitHandler]);
 
-  return {
-    panels,
-    createSubmitHandlerForPanel,
-    panelProperties: {
+  return useMemo(
```

Comment:
What is causing the rerender on element drag? 

You can try toggling `Record why each component rendered while profiling.` under `Settings > Profiler` in React Profiler or use `useWhyDidYouUpdate()`.

Reply:
Yeah I used both methods already, which led to this PR in the first place.

Actually quite few causes:

* `selectedElements`
* `submittedSelectedElements`
* `selectedElementAnimations`
* `pushUpdate`
* `pushUpdateForObject`

This PR fixes the `selectedElementAnimations` case for sure, but the others seem unavoidable, since the selected element changes its position and thus these variables (referencing or updating the current element) change.